<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoGry</class>
 <widget class="QDialog" name="OknoGry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="viewPlansza">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1015</width>
     <height>1015</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="player1Label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="player1Score">
   <property name="geometry">
    <rect>
     <x>1219</x>
     <y>10</y>
     <width>51</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="player2Label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>60</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="player2Score">
   <property name="geometry">
    <rect>
     <x>1219</x>
     <y>60</y>
     <width>51</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="player3Label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>110</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="player3Score">
   <property name="geometry">
    <rect>
     <x>1219</x>
     <y>110</y>
     <width>51</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="player4Label">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>160</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="player4Score">
   <property name="geometry">
    <rect>
     <x>1219</x>
     <y>160</y>
     <width>51</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRzut">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>300</y>
     <width>230</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>Rzut Kostką</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelZiarno">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>420</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnZakoncz">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>780</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Zakończ</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelJestTura">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>220</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jest tura gracza 1.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelZwyciestwo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1011</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>550</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Zwyciestwo test</string>
   </property>
  </widget>
  <zorder>player1Label</zorder>
  <zorder>player1Score</zorder>
  <zorder>player2Label</zorder>
  <zorder>player2Score</zorder>
  <zorder>player3Label</zorder>
  <zorder>player3Score</zorder>
  <zorder>player4Label</zorder>
  <zorder>player4Score</zorder>
  <zorder>btnRzut</zorder>
  <zorder>labelZiarno</zorder>
  <zorder>viewPlansza</zorder>
  <zorder>btnZakoncz</zorder>
  <zorder>labelJestTura</zorder>
  <zorder>labelZwyciestwo</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
